.github/workflows/release.yaml

```diff
@@ -108,7 +114,7 @@ jobs:
           ${{ env.DOCKER_TAG }}
           ${{ env.LATEST_TAG }}
         build-args: |
-          BASE_IMAGE=mosaicml/pytorch:2.4.0_cu124-python3.11-ubuntu20.04
+          BASE_IMAGE=mosaicml/llm-foundry:2.4.0_cu124-latest
```

Comment:
same here

Reply:
is this cached....?

Reply:
closing this to keep everything in one thread